supervisely/api/video/video_annotation_api.py

```diff
@@ -280,15 +288,71 @@ async def download_async(
             loop = sly.utils.get_or_create_event_loop()
             ann_info = loop.run_until_complete(api.video.annotation.download_async(video_id))
         """
-        if video_info is None:
-            video_info = self._api.video.get_info_by_id(video_id)
+        return await self.download_bulk_async(
+            [video_id],
+            semaphore,
+            force_metadata_for_links,
+            integer_coords,
+            progress_cb,
```

Comment:
better to pass named args `video_ids=[video_id],`
(the same for pcd and volumes)